examples/server/server.cpp

```diff
@@ -196,6 +198,10 @@ void whisper_print_usage(int /*argc*/, char ** argv, const whisper_params & para
 }
 
 bool whisper_params_parse(int argc, char ** argv, whisper_params & params, server_params & sparams) {
+    if (const char * env_gpu_device = std::getenv("GPU_DEVICE")) {
```

Comment:
And perhaps this should be `WHISPER_ARG_DEVICE` to be consistent with [llama.cpp](https://github.com/ggml-org/llama.cpp/blob/933414c0b6f21af269bdb4fa2fa1b257b9c0fc53/common/arg.cpp#L1881).